src/EditorFeatures/Core/Implementation/Classification/ClassificationTypeDefinitions.cs

```diff
@@ -58,6 +58,12 @@ internal sealed class ClassificationTypeDefinitions
         [BaseDefinition(ClassificationTypeNames.ClassName)]
         internal readonly ClassificationTypeDefinition UserTypeRecordsTypeDefinition;
         #endregion
+        #region User Types - Record Structs
+        [Export]
+        [Name(ClassificationTypeNames.RecordStructName)]
+        [BaseDefinition(ClassificationTypeNames.StructName)]
```

Comment:
interesting base-definition.  i would honestly expect .FormatLanguage here.  but i suppose this won't hurt :)

Reply:
Looking at the history, for the corresponding record class case, it started as FormatLanguage, but then got changed to ClassName.
I have no opinion (first time touching this)

---
In reply to: [609208508](https://github.com/dotnet/roslyn/pull/52464#discussion_r609208508) [](ancestors = 609208508)